### DISCUSSION TITLE
Unable to download beatmaps via osu! Direct on Android

### DISCUSSION THREAD

**Discussion by Mip2406** [ppy/osu#16912] [Q&A]
whenever i try to download a beatmap via osu direct it would just give the error message "Beatmapset download failed!"

**Comment by smoogipoo** [comment]
https://github.com/ppy/osu/discussions/15188

**Reply by Mip2406** [reply]
> #15188

thats another problem... its not giving me any logs

**Reply by frenzibyte** [reply]
Haven't you mentioned that it was due to your device not allowing file permissions for osu!? we require logs to give you further assistance on the matter.
